src/modding/ModManager.cs

```diff
@@ -903,6 +1617,39 @@ private void CloseModInfoPopup()
         modFullInfoPopup.Hide();
     }
 
+    private void OnCheckPressed()
+    {
+        GUICommon.Instance.PlayButtonPressSound();
+
+        if (enabledMods is null)
+        {
+            return;
+        }
+
+        // Checks if there is anything that is going to be loaded first
+        if (enabledMods.Count <= 0)
+        {
+            return;
+        }
+
+        var checkResult = ModLoader.IsValidModList(enabledMods);
```

Comment:
Refer to my earlier comments, but I think that empty mod list should be considered valid to reduce the amount of logic needed here.